pipeline.py

```diff
@@ -69,31 +88,94 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
+        self.corpus_manager = corpus_manager
         pass
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        articles = self.corpus_manager.get_articles().values()
+        for article in articles:
+            raw_text = article.get_raw_text()
+            tokens = self._process(raw_text)
+            cleaned_tokens = []
+            single_tagged_tokens = []
+            for token in tokens:
+                cleaned_tokens.append(token.get_cleaned())
+                single_tagged_tokens.append(token.get_single_tagged())
+            article.save_as(' '.join(cleaned_tokens), 'cleaned')
+            article.save_as(' '.join(single_tagged_tokens), 'single_tagged')
 
     def _process(self, raw_text: str):
         """
         Processes each token and creates MorphToken class instance
         """
-        pass
+        cleaned_text = re.sub(r"[^а-яА-Я\s]", "", raw_text)
+        analyzed_cleaned_text = Mystem().analyze(cleaned_text)
+        tokens = []
+        for token in analyzed_cleaned_text:
+            if 'analysis' not in token:
```

Comment:
is there a way to join all these conditions?

Reply:
or even maybe a way to just get it in one line? this is not a mandatory requirement, but at least write down right here in comments what ways do you see to shorten the logic?